docs/install/kubernetes.md

```diff
@@ -147,14 +157,15 @@ When the installation is complete, the helm command prints the URLs and Ports of
 
 ## Installing the Full Version
 
-To install the full version of the chart, use the following command:
+To install the full version of the chart first follow the instructions of installing the light version up until the helm
```

Comment:
To install the full version of the chart, follow the instructions of installing the light version up to an including creating a secret with your docker-registry, and then continue with: